vm/src/protocol/mapping.rs

```diff
@@ -45,7 +45,7 @@ impl PyMappingMethods {
         ];
 
         fn length(mapping: &PyMapping, vm: &VirtualMachine) -> PyResult<usize> {
-            crate::types::slot_length(mapping.obj, vm)
+            crate::types::len_wrapper(mapping.obj, vm)
```

Comment:
Could this be called through the protocol interface? It seems it'll go through the mro resolution inside python side, doesn't it?

Reply:
Oh is it for generic cases? Ok I see